src/App.tsx

```diff
@@ -63,7 +63,7 @@ const App: React.FunctionComponent = () => {
         <Route path="/authorization" element={<AuthorizationPage />} />
         <Route path="/followers" element={<Followers />} />
         <Route path="/following" element={<Following />} />
-        <Route path="/trends" element={<Trends />} />
+        <Route path="/:trends" element={<Trends />} />
```

Comment:
what? no it's not what I've asked

Reply:
you need to use hook to get path, not param

Reply:
sorry. Changed